tracing/src/main/java/org/axonframework/extensions/tracing/TracingQueryGateway.java

```diff
@@ -109,9 +116,27 @@ public static Builder builder() {
                                                                      ResponseType<U> updateResponseType,
                                                                      SubscriptionQueryBackpressure backpressure,
                                                                      int updateBufferSize) {
-        return delegate.subscriptionQuery(
-                queryName, query, initialResponseType, updateResponseType, backpressure, updateBufferSize
-        );
+
+        QueryMessage qry = new GenericQueryMessage(query, queryName, initialResponseType);
```

Comment:
The same "is this a message?" check should be done here.

Reply:
Updated as well.